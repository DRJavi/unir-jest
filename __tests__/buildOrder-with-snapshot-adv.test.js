'use strict';

const fixtures = require('../__fixtures__/buildOrder');
const orders = require('../');

it('builds correctly (with advanced snapshot)', () => {  
  const result = orders.buildOrder(fixtures.listItems);
 
  expect(result.total).toBeCloseTo(97.9, 5);
  expect(result).toMatchSnapshot({
    total: expect.any(Number)
  });
});
