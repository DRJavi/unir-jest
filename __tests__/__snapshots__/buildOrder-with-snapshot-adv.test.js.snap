// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`builds correctly (with advanced snapshot) 1`] = `
Object {
  "items": Array [
    "Item 1",
    "Item 2",
    "Item 5",
  ],
  "total": Any<Number>,
}
`;
